api/turing/api/appcontext.go

```diff
@@ -104,6 +104,7 @@ func NewAppContext(
 		}
 		appContext.AlertService = service.NewGitlabOpsAlertService(db, gitlabClient, cfg.AlertConfig.GitLab.ProjectID,
 			cfg.AlertConfig.GitLab.Branch, cfg.AlertConfig.GitLab.PathPrefix)
+		service.AlertPlaybookURL = cfg.AlertConfig.PlaybookURL
```

Comment:
This feels a bit ad-hoc to me. I guess `AlertPlaybookURL` is not a part of `gitlabOpsAlertService` because it is common to other types of alert service implementations? Can setting this be made the responsibility of the constructor in the Alert Service? So in the appcontext init we just do:
```
if cfg.AlertConfig.Enabled {
    appContext.AlertService, err := serice.NewAlertService(db, cfg.AlertConfig)
}
```
I guess this will also be closer to how we initialize the experiment managers in `NewExperimentsService`. Wdyt?

Reply:
yeah sounds good. I will update that in another PR